src/main/types.d.ts

```diff
@@ -30,9 +30,11 @@ declare interface Row {
   readonly notes: Text
 }
 
+type RoleTodoIdMap = { [key: string]: string };
```

Comment:
We should probably add a comment here to document what the key and value are for this map